subprojects/test-runner/device-provider/impl/src/main/kotlin/com/avito/android/runner/devices/internal/kubernetes/KubeContainer.kt

```diff
@@ -0,0 +1,84 @@
+package com.avito.android.runner.devices.internal.kubernetes
```

Comment:
Those classes are good candidates for separate module: `kubernetes-extensions`

Reply:
we have such module, see `:kubernetes`, 
but `KubePod` has some internal logic, it assumes that we always have 1 container per pod